packages/core/src/lambda/utils.ts

```diff
@@ -13,8 +13,6 @@ import { CloudFormationClient } from '../shared/clients/cloudFormationClient'
 import { LambdaClient } from '../shared/clients/lambdaClient'
 import { getFamily, getNodeMajorVersion, RuntimeFamily } from './models/samLambdaRuntime'
 import { getLogger } from '../shared/logger'
-import { ResourceFetcher } from '../shared/resourcefetcher/resourcefetcher'
-import { CompositeResourceFetcher } from '../shared/resourcefetcher/compositeResourceFetcher'
```

Comment:
Future idea: I noticed long ago that `CompositeResourceFetcher` is re-inventing "first-class functions".  It's not needed in js/ts.

Reply:
Yeah I was thinking about just removing it. CompositeResourceFetcher was only ever used in that specific case